### ISSUE TITLE
How do I compile fc separately

### ISSUE THREAD

**Issue by danielAnth** [EOSIO/fc#51]
How to solve the fc separate compile time error?

**Comment by cc32d9** [comment]
Here I integrated it successfully:  https://github.com/EOSChronicleProject/eos-chronicle

**Comment by spoonincode** [comment]
I believe this has been fixed since #98. Compiling fc standalone should work (though not very useful I wouldn't think). Also, using fc in another project should be easier.

For a local project that is using fc, my CMakeLists.txt simply looks like
```
cmake_minimum_required(VERSION 3.10)
project(cms)

set(CMAKE_CXX_STANDARD 17)

add_subdirectory(fc)
# ...
target_link_libraries(cmstest fc Boost::unit_test_framework libcms)
```

One does need to make sure c++17 is enabled before including fc (it will try to set c++17 otherwise)
